base.cfg

```diff
@@ -18,6 +18,7 @@ parts =
 develop = .
 sources-dir = extras
 auto-checkout =
+    plone.volto
```

Comment:
We don't really need to tie this to the plone.volto change. So I'd remove this for the merge.